jenkins/slave-base/Dockerfile.centos7

```diff
@@ -1,4 +1,7 @@
-FROM openshift/jenkins-slave-base-centos7
+# BuildConfig (../ocp-config/bc.yml) overrides FROM instruction
```

Comment:
Shouldn't we add the config there then?

Reply:
yes, partially it's already there.
https://github.com/opendevstack/ods-core/blob/master/jenkins/ocp-config/bc.yml#L121-L124

Reply:
Yes sorry, I should have commented for the master image. The slave image config is already there. I would have actually preferred if your proposed solution would also be just one variable for both image and tag - I don't see the value of having both?